src/packet_analysis/protocol/udp/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
-
-include(ZeekPlugin)
-
-include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
-
-zeek_plugin_begin(Zeek UDP)
-zeek_plugin_cc(UDP.cc UDPSessionAdapter.cc Plugin.cc)
-zeek_plugin_bif(events.bif)
-zeek_plugin_end()
+zeek_add_plugin(
+    Zeek
```

Comment:
I think this one has to stay as `Zeek` or other things break, but we should probably test that. Same with the TCP one as well.